README.md

```diff
@@ -35,15 +35,42 @@ Like all standard Jackson modules (libraries that implement Module interface), r
 ```java
 ObjectMapper mapper = new ObjectMapper();
 mapper.registerModule(new Jdk8Module());
+// Or, the more fluent version: ObjectMapper mapper = new ObjectMapper().registerModule(new Jdk8Module());
 ```
 
 after which functionality is available for all normal Jackson operations:
 you can read JSON into supported JDK8 types, as well as write values of such types as JSON, so that for example:
 
 ```java
-// TODO: real example
-Optional<String> str = ...;
-String json = mapper.writeValueAsString(str);
+class Contact {
+    @JsonProperty
```

Comment:
I personally don't rely on private field discovery either, so I changed it (to what I consider a good practice). And it also promotes a nice feature of Jackson (using getters instead of changing code with annotations).